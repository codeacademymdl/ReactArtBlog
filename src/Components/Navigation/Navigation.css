.nav {
    padding: 10px 80px;
    background: #000;
    color: #fff;
    display: flex;
    justify-content: space-between;
    align-items: flex-end;
    
  }

.heading{
 margin-left:5%;

}
  .nav div h2{

    margin-top:-20px;
    font-style: italic;
    font-size:1em;
  }
  .navLink{

    width:40%;
    display:flex;
    justify-content: space-between;
    min-width:20%;
  }
  .navLink a {

    color:white;
    text-decoration: none; 

  }

  /* .navLink a:nth-child(2){
      margin:0 100px;
  } */