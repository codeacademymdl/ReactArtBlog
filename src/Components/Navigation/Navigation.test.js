import React from 'react';

import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Navigation from './Navigation'

describe('Navigation', () => {
    it('renders correctly', () => {
        const tree = renderer
          .create(<Navigation/>)
          .toJSON();
        expect(tree).toMatchSnapshot();
      });
});

