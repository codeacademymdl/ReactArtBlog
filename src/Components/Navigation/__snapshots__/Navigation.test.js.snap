// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation renders correctly 1`] = `
<div
  className="nav"
>
  <div
    className="heading"
  >
    <h1>
      The Artifact
    </h1>
    <h2>
      Culture & Art Blog
    </h2>
  </div>
  <div
    className="navLink"
  >
    <a
      href="#!"
    >
      Blog
    </a>
    <a
      href="#!"
    >
      About
    </a>
    <a
      href="#!"
    >
      Contact
    </a>
  </div>
</div>
`;
